shuup/campaigns/admin_module/__init__.py

```diff
@@ -25,7 +25,7 @@ def get_urls(self):
         basket_campaign_urls = get_edit_and_list_urls(
             url_prefix="^campaigns/basket",
             view_template="shuup.campaigns.admin_module.views.BasketCampaign%sView",
-            name_template="basket_campaigns.%s",
+            name_template="basket_campaign.%s",
```

Comment:
In case you are changing this you also need to change url references in `get_menu_entries()`